kumascript/macros/LearnSidebar.ejs

```diff
@@ -1882,6 +1895,25 @@ var text = mdn.localStringMap({
         </ol>
     </details>
   </li>
+<li><a href="<%=baseURL%>Performance"><strong><%=text['Performance_—_Making_websites_fast_and_responsive']%></a></strong></li>
```

Comment:
The key here: `Performance_—_Making_websites_fast_and_responsive` isn't matching the string map at https://github.com/mdn/yari/pull/8007/files#diff-f42bbd8a5a864e6d8d40c7f6fc6654d123d9697244248a6e13556fe975d53aa7R171, so we get no link text displayed.

Reply:
I guess this should be:

```suggestion
<li><a href="<%=baseURL%>Performance"><strong><%=text['Performance']%></a></strong></li>
```